src/goto-symex/symex_target.h

```diff
@@ -17,6 +17,9 @@ Author: Daniel Kroening, kroening@kroening.com
 class ssa_exprt;
 class symbol_exprt;
 
+/// Base class for symbolic execution. Intended as a position identifier in the
+/// context of symbolic execution (similar as goto_programt::const_target is an
```

Comment:
:pick: Typo - `similar to goto_programt::const_target which is ..`?

Reply:
Done.